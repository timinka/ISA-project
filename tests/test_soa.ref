2024-10-20 16:41:07 192.168.0.89 -> 192.168.0.1 (Q 1/0/0/0)
2024-10-20 16:41:07 192.168.0.1 -> 192.168.0.89 (R 1/1/0/0)
