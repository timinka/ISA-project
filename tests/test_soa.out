Timestamp: 2024-11-16 16:41:07
SrcIP: 192.168.0.89
DstIP: 192.168.0.1
SrcPort: UDP/57257
DstPort: UDP/53
Identifier: 0x0004
Flags: QR=0, OPCODE=0, AA=0, TC=0, RD=1, RA=0, AD=0, CD=0, RCODE=0

[Question Section]
vut.cz. IN SOA
====================
Timestamp: 2024-11-16 16:41:07
SrcIP: 192.168.0.1
DstIP: 192.168.0.89
SrcPort: UDP/53
DstPort: UDP/57257
Identifier: 0x0004
Flags: QR=1, OPCODE=0, AA=0, TC=0, RD=1, RA=1, AD=0, CD=0, RCODE=0

[Question Section]
vut.cz. IN SOA

[Answer Section]
vut.cz. 289 IN SOA rhino.cis.vutbr.cz. hostmaster.vutbr.cz. 2024110800 28800 7200 1814400 300
====================
